accepted/2021/net6.0-tfms/net6.0-tfms.md

```diff
@@ -149,6 +152,33 @@ not generate a warning. This reasoning results in these precedence rules:
 1. `netstandard2.1` – `1.0`
 1. `net4.x` – `1.0` ([NU1701] warning)
 
+**net6.0-android**
+
+1. `net6.0-android`
+1. `net6.0`
+1. `monoandroid12.0` - `1.0` (no warning)
+1. `net5.0`
+1. `netcoreapp3.1` – `1.0`
+1. `netstandard2.1` – `1.0`
+1. `net4.x` – `1.0` (warning)
+
+The currently supported `monoandroid` TFMs are:
+
+* `monoandroid1.0`
+* `monoandroid4.4`
+* `monoandroid4.4.87`
+* `monoandroid5.0`
+* `monoandroid5.1`
+* `monoandroid6.0`
+* `monoandroid7.0`
+* `monoandroid7.1`
+* `monoandroid8.0`
+* `monoandroid8.1`
+* `monoandroid9.0`
+* `monoandroid10.0`
+* `monoandroid11.0`
+* `monoandroid12.0`
```

Comment:
Do you want this to automatically support any future `monoandroid` releases? (that's the default. Just wondering if you want to stop that)

Reply:
I think supporting all newer versions is a good idea. We will probably have a future `monandroid13.0` sometime in 2022, depending on what Google chooses the next OS version to be.

Reply:
I assume NuGet doesn't need to know any of these versions, correct? We'd just fall back to the latest `monoandroidX.Y`, correct?

Reply:
I don't think they need to account for specific versions, as long as NuGet picks the highest `monoandroid` it finds in a given NuGet package.